sdk-setup/src/mb2

```diff
@@ -31,9 +31,9 @@ short_usage() {
     cat <<EOF
 usage: $ME [-t <target>] [-s <specfile>] [-d <device>] [-p <projectdir>]
            [-f <folder>] [-i] [-P] [-x]
-           build [-d] [-j <n>] [<args>] | qmake [<args>] | make [<args>] | ssh <args>
+           build [-e] [-d] [-j <n>] [<args>] | qmake [<args>] | make [<args>] | ssh <args>
```

Comment:
Why ```-e``` not ```-p``` (lower case).  Could we use same flags as there are in rpmbuild?

See:
http://www.rpm.org/max-rpm-snapshot/ch-rpm-b-command.html